package.json

```diff
@@ -62,11 +62,11 @@
   "dependencies": {
     "@types/vexflow": "^1.2.33",
     "jspdf-yworks": "^2.1.1",
-    "jszip": "3.4.0",
+    "jszip": "^3.4.0",
     "loglevel": "^1.6.8",
     "svg2pdf.js": "^1.5.0",
     "typescript-collections": "^1.3.3",
-    "vexflow": "3.0.4"
+    "vexflow": "^3.0.4"
```

Comment:
using the caret for the Vexflow version is not a good idea. Vexflow ^3.0.4 means any Vexflow version at or above 3.0.4, below 4.0 (compatible with 3.0.4), arbitrarily the most recent one when you last performed npm install.
There are often breaking changes in Vexflow updates, so each version should be tested and updated to precisely and carefully, without a caret.

Reply:
by the way, we are using 1.2.93 right now, so the pull request shouldn't change this. It would be nice if you could merge opensheetmusicdisplay/develop. (not the develop in your fork)